### DISCUSSION TITLE
hostname/ip does not match certificate's altnames  127.0.01

### DISCUSSION THREAD

**Discussion by alanfg120** [rabbitmq/tls-gen#53] [General]
I would like to know how I generate a certificate that accepts the hostname 127.0.0.1 when connecting through this IP with the basic type certificates I get the error

hostname/ip does not match certificate's altnames  127.0.01

**Comment by michaelklishin** [comment]
Consider using Discussions and not issues for questions.

All profiles already [list `localhost`](https://github.com/rabbitmq/tls-gen/blob/main/basic/openssl.cnf#L85) as one of the alternative names. `127.0.0.1` is not a hostname.
